contrib/mysql.sql

```diff
@@ -1,2 +0,0 @@
-DROP DATABASE IF EXISTS gitea;
```

Comment:
Phew, good that apparently no one ever used this script on a production instance :relieved: